client/src/components/reviewStars.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+let ReviewStars = (props) => {
+  if (props.include === true) {
+    return (
+      <div className="review-stars">
+        {
+          Array.from({length: 5}).map( (undef, i) => {
```

Comment:
I didn't know you could do this, I usually just do `[...Array.from(5)]`. Good to know!